### DISCUSSION TITLE
x-reactions深度监听数组

### DISCUSSION THREAD

**Discussion by boyrest** [alibaba/formily#1864] [Q&A 2.x]
怎么实现对数组的深度依赖
'x-reactions': [
      {
        dependencies: ['array'],
        fulfill: {
          state: {
            'visible': '{{console.log(123)}}',
          },
        },
      },
    ],
比如：
[{name:‘name1’}]=>[{name:‘name12323’}] 这种情况下触发依赖响应

**Comment by janryWang** [comment]
自己先搜索一下问题再提问题
